build.sbt

```diff
@@ -19,7 +22,7 @@ lazy val root = (project in file("."))
   .aggregate(api, frontend, backend)
   
 lazy val frontend = (project in file("frontend"))
-    .enablePlugins(PlayScala, BuildInfoPlugin)
+    .enablePlugins(PlayScala, BuildInfoPlugin, JavaAppPackaging)
```

Comment:
I think this is not really needed, as play already adds the server_archetype. However we can keep this to make things more clear.